web/src/components/map/Map/HexagonExploreMap.tsx

```diff
@@ -353,15 +361,44 @@ export default function HexagonExploreMap({
               anchor={[100, 100]}
               className="pigeon-center-buttons"
             >
-              <button
-                className="pigeon-center-view"
-                onClick={() => {
-                  store.emit(new RecenterExplore());
-                }}
-              >
-                <IoStorefrontSharp />
-              </button>
+              {viewMode == 'map' &&
+                  <button
+                    className="pigeon-full-map"                
+                    onClick={() =>
+                        store.emit(
+                          new UpdateExploreViewMode(ExploreViewMode.BOTH),
+                        )
+                      }
+                  >
+                  
+                  <IoContract />
+                </button>
+              }
+              {viewMode != 'map' &&
+                  <button
+                    className="pigeon-full-map"                
+                    onClick={() =>
```

Comment:
@watchoutfreedom I dont see any of this buttons. it should up in the explore view no?